api/room/models.py

```diff
@@ -10,6 +10,7 @@
 from api.events.models import Events  # noqa: F401
 from api.response.models import Response  # noqa: F401
 from api.tag.models import Tag  # noqa: F401
+from api.devices.models import Devices # noqa F4
```

Comment:
Kindly share the reason for introducing this import.

Reply:
Without that import when you import the Room model and try to create a query you get an error that `Devices` not recognized.

Reply:
Is it possible to do away with the `#noqas`